dev/integration_tests/high_bitrate_images/lib/main.dart

```diff
@@ -55,19 +57,62 @@ class _SdfCanvasState extends State<SdfCanvas> {
         _shader = shader;
       });
     });
-    _loadSdfImage().then((ui.Image image) {
-      setState(() {
-        _sdfImage = image;
-      });
-    });
+    switch (gTargetPixelFormat) {
+      case ui.TargetPixelFormat.rgbaFloat32:
+        _loadRGBA32FloatSdfImage().then((ui.Image image) {
+          setState(() {
+            _sdfImage = image;
+          });
+        });
+      case ui.TargetPixelFormat.rFloat32:
+        _loadR32FloatSdfImage().then((ui.Image image) {
+          setState(() {
+            _sdfImage = image;
+          });
+        });
+      case ui.TargetPixelFormat.dontCare:
+        assert(false);
+    }
   }
 
   Future<ui.FragmentShader> _loadShader() async {
     final ui.FragmentProgram program = await ui.FragmentProgram.fromAsset('shaders/sdf.frag');
     return program.fragmentShader();
   }
 
-  Future<ui.Image> _loadSdfImage() async {
+  Future<ui.Image> _loadRGBA32FloatSdfImage() async {
```

Comment:
The names and implementation seem backwards here. The names are rgba32 then r32, but the implementations are r32 then rgba32?